src/index.js

```diff
@@ -1 +1,29 @@
 import './style.css';
+
+const todo = [
+  {
+    description: 'wash the dishes',
+    completed: false,
+  },
+  {
+    description: 'Complete todo list tasks',
+    completed: false,
+  },
```

Comment:
Kindly add an index property to your objects. It will help you to differentiate your tasks making removing or editing them easier. :nerd_face: :nerd_face:

Reply:
Thanks @Santiago220991 for the review :clap: . Done :exclamation: